spec/models/service_ansible_playbook_spec.rb

```diff
@@ -0,0 +1,92 @@
+describe(ServiceAnsiblePlaybook) do
+  let(:tower_job)      { FactoryGirl.create(:ansible_tower_job) }
+  let(:tower_job_temp) { FactoryGirl.create(:ansible_configuration_script) }
+  let(:basic_service)  { FactoryGirl.create(:service_ansible_playbook, :options => dialog_options) }
+
+  let(:loaded_service) do
+    service_template = FactoryGirl.create(:service_template)
+    service_template.resource_actions.build(:action => 'Provision', :configuration_template => tower_job_temp)
+    service_template.save!
+    FactoryGirl.create(:service_ansible_playbook, :options => provision_options, :service_template => service_template)
+  end
+
+  let(:executed_service) do
+    basic_service.tap do |service|
+      allow(service).to receive(:job).with('Provision').and_return(tower_job)
+    end
+  end
+
+  let(:dialog_options) do
+    {
+      :dialog => {
+        'dialog_hosts'      => 'host1;host2',
```

Comment:
Should probably follow Ansible's lead on this.  See the  notes at the bottom of this page: http://docs.ansible.com/ansible/intro_patterns.html

> Note
With the exception of version 1.9, you can use ‘,’ instead of ‘:’ as a host list separator. The ‘,’ is preferred specially when dealing with ranges and ipv6.

> Note
As of 2.0 the ‘;’ is deprecated as a host list separator.

Looks like we should use a comma as the separator.